pep-0249.txt

```diff
@@ -201,8 +201,8 @@ exceptions or subclasses thereof:
 
 This is the exception inheritance layout::
 
```

Comment:
This currently renders as:
![image](https://user-images.githubusercontent.com/9087854/188290527-25e5991a-3bdd-4c84-97a6-f45b823de49c.png)

I don't imagine you want that, so I'd suggest changing to "text" (no highlighting) and removing the underscores from the footnotes, as they won't be interpreted as reStructuredText.

```suggestion
This is the exception inheritance layout:

.. code-block:: text
```